xgcm/transform.py

```diff
@@ -62,12 +67,19 @@ def interp_1d_linear(
         Option to bypass logic to flip data if monotonically decreasing along the axis.
         This will improve performance if True, but the user needs to ensure that values
         are increasing alon the axis.
+    logarithmic : bool, optional
+        Apply a logarithmic transform to theta and target_theta_levels so that the
+        interpolation is done in logarithmic, rather than linear, space. In turn, theta
+        must be positive. Defaults to False.
 
     Returns
     -------
     phi_interp : array
         Array of shape (..., m) of phi interpolated to theta isosurfaces.
     """
+    if logarithmic:
```

Comment:
Love it. There were literally only 4 lines needed to implement this option! Everything else are docstrings!